fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -585,6 +587,9 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     @VariableMgr.VarAttr(name = TRANSMISSION_COMPRESSION_TYPE)
     private String transmissionCompressionType = "LZ4";
 
+    @VariableMgr.VarAttr(name = TRANSMISSION_ENCODE_LEVEL)
```

Comment:
how many encode levels could we have? and what's the meaning for each level?

Reply:
encode_level & 1  for encode fixed_column
encode_level & 2 for encode binary_column

Reply:
Add this info to comment and document.